include/zendoo_mc.h

```diff
@@ -766,19 +766,25 @@ extern "C" {
 
     /*
      * Perform batch verification of all the proofs added to `batch_verifier`.
+     * If `prioritize` is set to true, pauses other batch verifications (if any)
+     * happening in other threads, asap, to give priority to this one.
      */
     ZendooBatchProofVerifierResult* zendoo_batch_verify_all_proofs(
         const sc_batch_proof_verifier_t* batch_verifier,
+        bool prioritize,
         CctpErrorCode* ret_code
     );
 
     /*
      * Perform batch verification of the proofs added to `batch_verifier` whose id is contained in `ids_list`.
+     * If `prioritize` is set to true, pauses other batch verifications (if any) happening in other threads,
+     * asap, to give priority to this one.
      */
     ZendooBatchProofVerifierResult* zendoo_batch_verify_proofs_by_id(
         const sc_batch_proof_verifier_t* batch_verifier,
         const uint32_t* ids_list,
         size_t ids_list_len,
+        bool prioritize,
         CctpErrorCode* ret_code
     );
 
```

Comment:
As far as I've understood, calling the batch verification with the `prioritize` flag set to `true` makes only "normal priority" threads be stopped, while "high priority" threads are unaffected.

If this is the case, I would specify such behavior in the comments for clarity's sake.